Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/CosmeticFiltersScriptHandler.swift

```diff
@@ -83,6 +87,19 @@ class CosmeticFiltersScriptHandler: TabContentScript {
           }
         }
         
+        // Based on what we block we want to show the youtube warning
+        // TODO: @JS Figure out which selectors should trigger this warning
+        if !ShieldPreferences.hasSeenAntiAdBlockWarning.value {
```

Comment:
So this alert will be shown every time user open youtube on a new tab page even user dismisses the pop-over. Only way not to see this is to press one of the buttons now.

I believe this requirement is wrong and should ne working this way, where we harass the user with this new pop-over.

Reply:
No, I just haven't figured out how to check if the anti-adblock warning was seen yet. But this should only appear once that condition is met. But I can't reproduce the anti-adblock warning so I'm a little bit stuck, I removed that condition from this PR for now and will add it as a separate PR. 
And once the user choses an option, yes or no, then it won't appear ever again.